### PR TITLE
feat(eslint-config-angular): add import rules

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/eslint-config-angular/internal/import.js [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+  plugins: ['simple-import-sort', 'eslint-plugin-import'],
   settings: {
     'import/parsers': { '@typescript-eslint/parser': ['.ts'] },
     'import/resolver': { 'eslint-import-resolver-typescript': true },
   },
   rules: {
+    'simple-import-sort/imports': 'error',
+    'simple-import-sort/exports': 'error',
+    'import/first': 'error',
+    'import/no-default-export': 'error',
```
Comment (IKatsuba) [inline packages/eslint-config-angular/internal/import.js:12]: это че это?

Comment (splincode) [reply packages/eslint-config-angular/internal/import.js:12]: For many years, in different projects, I ran into problems with default export. And many people still think that these are bad practices.  I consider default exports an anti-pattern.

@SuperOleg39 @IKatsuba what do you think?

Reference:
https://basarat.gitbook.io/typescript/main-1/defaultisbad
https://www.reddit.com/r/javascript/comments/hg3wl4/is_using_default_exports_considered_a_bad/
https://tunein.com/podcasts/Technology-Podcasts/Web-Standards-p960847/?topicId=164594951
https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad

Comment (IKatsuba) [reply packages/eslint-config-angular/internal/import.js:12]: Вообще весь React мир живет на экспортах по дэфолту. И много библиотек (особенно неподдерживаемых) до сих пор приходиться использовать именно так. кажется, что записывать это в ошибку, даже для Angular мира, ту мач.

Comment (splincode) [reply packages/eslint-config-angular/internal/import.js]: updated

**Review by IKatsuba — APPROVED** [review]

File: packages/eslint-config-angular/internal/import.js [hunk 2]
```diff
@@ -1,10 +1,26 @@
 module.exports = {
   parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint', 'simple-import-sort', 'eslint-plugin-import'],
```
Comment (SuperOleg39) [inline packages/eslint-config-angular/internal/import.js:3]: @typescript-eslint плагин уже есть тут https://github.com/TinkoffCreditSystems/linters/blob/master/packages/eslint-config/internal/typescript.js

**Review by SuperOleg39 — APPROVED** [review]
LGTM

**Pull request merged**